python_on_whales/components/compose/cli_wrapper.py

```diff
@@ -193,6 +195,7 @@ def create(
             no_build: Don't build an image, even if it's missing.
             no_recreate: If containers already exist, don't recreate them.
                 Incompatible with `force_recreate=True`.
+            pull: Pull image before running (“always”|”missing”|”never”|”build”).
```

Comment:
```suggestion
            pull: Pull image before running ("always"|"missing"|"never"|"build").
```
Let's use normal quotes

Reply:
Sounds good to me, although I copied it from the `up()` docs for consistency...